### PR TITLE
Add project status section to main page

### PR DESCRIPTION
**Pull request by ubruhin**
See https://github.com/LibrePCB/LibrePCB/issues/222

Rendered: https://docs.librepcb.org/v/add_status_section/#projectstatus

@dbrgn Any thoughts?

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -22,5 +22,39 @@ image:img/screenshot.png[alt="screenshot"]
 * Multi-PCB feature (different PCB variants of the same schematic)
 * Automatic netlist synchronisation between schematic and board
 
-*You want to give LibrePCB a try?* +
+
+[#projectstatus]
+== Project Status
+
+LibrePCB is still under development and many things are not yet
+working as expected. So here an overview about the project status:
```
Comment (dbrgn) [inline README.adoc]: "here's" or "here is"

File: README.adoc [hunk 2]
```diff
@@ -22,5 +22,39 @@ image:img/screenshot.png[alt="screenshot"]
...
+
+=== What's not working (or even missing)
+
+* Schematic editor:
+** No hierarchical sheets
+** No buses
+** No BOM export
+* Board editor:
+** No airwires
+** No texts (e.g. on silkscreen)
```
Comment (dbrgn) [inline README.adoc]: texts -> text

File: README.adoc [hunk 3]
```diff
@@ -22,5 +22,39 @@ image:img/screenshot.png[alt="screenshot"]
...
+** No planes (filled copper areas)
+** No 3D board viewer
+** *Generally very limited functionality and many bugs because
+   it's still provisional!*
```
Comment (rnestler) [inline README.adoc]: I would replace provisional with "work in progress" or "under development".

**Comment by rnestler** [comment]
A bit unrelated, but should we create tracking issues on GitHub for those pending items? We could then link to them from the documentation.

File: README.adoc (hunk 2 above)
Comment (ubruhin) [reply README.adoc]: Sure? "No airwires" and "No planes" is also plural, so why use singular for "text"?

File: README.adoc (hunk 1 above)
Comment (ubruhin) [reply README.adoc]: done

File: README.adoc (hunk 3 above)
Comment (ubruhin) [reply README.adoc]: done

**Comment by ubruhin** [comment]
> A bit unrelated, but should we create tracking issues on GitHub for those pending items?

I added some links to already existing issues or to the wishlist.

File: README.adoc (hunk 2 above)
Comment (dbrgn) [reply README.adoc]: I think it's a bit unusual to use "texts" in plural, similar to "information". What do you think @rnestler?

Comment (ubruhin) [reply README.adoc]: I would say in this context "text" means an object on the board, and there can be multiple of them, so the plural form is fine here. See also:
- https://en.wiktionary.org/wiki/texts#English
- https://www.wordhippo.com/what-is/the-plural-of/text.html

Comment (rnestler) [reply README.adoc]: As a compromise I'd suggest "No text objects"

Comment (ubruhin) [reply README.adoc]: too late ;)

**Pull request merged**